ain/ain.py

```diff
@@ -0,0 +1,40 @@
+from ain.provider import Provider
+from ain.net import Network
+from ain.wallet import Wallet
+from ain.db import Database
+from ain.utils import ecSignTransaction, getTimestamp
+
+
+class Ain:
+    def __init__(self, providerUrl: str, chainId: int = 0):
+        self.provider = Provider(self, providerUrl)
+        self.chainId = chainId
+        self.net = Network(self.provider)
+        self.wallet = Wallet(self, self.chainId)
+        self.db = Database(self, self.provider)
+
+    # TODO : specify type `TransactionInput`
+
```

Comment:
Please try to fit in the Google Python style: https://google.github.io/styleguide/pyguide.html#35-blank-lines

Reply:
I think it doesn't provide 100% correct answers all the time, but it could be a good base of our own style conventions. Feel free to let us know if there are any parts you don't agree. @kriii